### ISSUE TITLE
it will report an error here in loss.backward()

### ISSUE THREAD

**Issue by CCJing14** [zhangmarvin/memo#4]
hello，i want to use memo to test coco. After calculating the marginal entropy, starting backpropagation to calculate the gradient, it will report an error here in loss.backward()
Error detected in SigmoidBackward0. Traceback of forward call that caused the error:
  File "test.py", line 636, in <module>
    test(opt.data,
  File "test.py", line 235, in test
    adapt_single(image, optimizer, batch_size, model, device)
  File "test.py", line 62, in adapt_single
    outputs,train_output = model(inputs)   # [batch_size, num_boxes, 85]
  File "miniconda3/envs/memo/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "miniconda3/envs/memo/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl
    return forward_call(*args, **kwargs)
  File "models/yolo.py", line 599, in forward
    return self.forward_once(x, profile)  # single-scale inference, train
  File "models/yolo.py", line 625, in forward_once
    x = m(x)  # run
  File "miniconda3/envs/memo/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "miniconda3/envs/memo/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl
    return forward_call(*args, **kwargs)
  File "models/yolo.py", line 153, in fuseforward
    y = x[i].sigmoid()
 (Triggered internally at ../torch/csrc/autograd/python_anomaly_mode.cpp:114.)
